pal/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+
+##
+# Create all libraries in the pal_libraries list for the chosen platform.
+##
+foreach(pal IN LISTS pal_libraries)
+    message(STATUS "creating library ${pal}[${${pal}}]")
+    add_library(${pal}
+        ${${pal}}
```

Comment:
so this is usually where sources would go. Is the double text replacement resolving a list of source files?

Reply:
Yep.

---
In reply to: [543622350](https://github.com/mamokarz/sample-multi-platform/pull/2#discussion_r543622350) [](ancestors = 543622350)